awx/ui_next/src/components/FormLayout/FormLayout.jsx

```diff
@@ -0,0 +1,44 @@
+import styled from 'styled-components';
+
+export const FormColumnLayout = styled.div`
+  width: 100%;
+  grid-column: 1 / 4;
+  display: grid;
+  grid-gap: 20px;
+  grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+
+  @media (min-width: 1210px) {
+    grid-template-columns: repeat(3, 1fr);
+  }
+`;
+
+export const FormFullWidthLayout = styled.div`
+  grid-column: 1 / -1;
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-gap: 20px;
+`;
+
+export const FormCheckboxLayout = styled.div`
+  display: flex;
+  justify-content: flex-start;
+  flex-wrap: wrap;
+
+  & > * {
+    margin-right: 30px;
```

Comment:
I would suggest deleting this `margin-right` and adding the rule:

```
& > :not(:first-child) {
    margin-left: 30px;
  }
```

this way there isn't extra margin after the last item (can cause weird line-wrapping issues)